README.md

```diff
@@ -7,63 +7,64 @@
-| 33  | [Service Binding](./033-service-binding.md) |
-| 32  | [Custom Authentication in Kafka Brokers ](./032-custom_authentication_in_kafka_brokers.md) |
-| 31  | [StatefulSet Removal](./031-statefulset-removal.md) |
-| 30  | [EnvVar Configuration Provider for Apache Kafka](./030-env-var-config-provider.md) |
-| 29  | [Adopt the Drain Cleaner utility](./029-adopt-the-drain-cleaner-utility.md) |
-| 28  | [Network Policy Generation Environment Variable](./028-network-policy-generation-environment-variable.md) |
-| 27  | [Kubernetes Configuration Provider for Apache Kafka](./027-kubernetes-config-provider.md) |
-| 26  | [Service Account patching](./026-service-account-patching.md) |
-| 25  | [Control Plane Listener](./025-control-plain-listener.md) |
-| 24  | [Adopt the Kafka Static Quota plugin](./024-adopt-the-kafka-static-quota-plugin.md) |
-| 23  | [Using Red Hat Universal Base Image 8 as the new Strimzi base image](./023-using-ubi8-as-base-image.md) |
-| 22  | [Feature Gates](./022-feature-gates.md) |
-| 21  | [Special repository for ST clients](./021-special-repository-for-st-clients-based-on-example-clients.md) |
-| 20  | [Rename the default branch of Strimzi GitHub repositories](./020-rename-default-branch-of-strimzi-github-repositories.md) |
-| 19  | [Restructure the installation files](./019-restruture-the-installation-files.md) |
-| 18  | [Use the admin-server REST API in strimzi-ui](./018-rest-admin-api.md) |
-| 17  | [Proxy-Based Kafka Per-Topic Encryption](./017-kafka-topic-encryption.md) |
-| 16  | [Modularizing Strimzi UI](./016-modularizing-strimzi-ui.md) |
-| 15  | [Kafka Connect build](./015-kafka-connect-build.md) |
-| 14  | [Move Container Images to Quay.io](./014-move-docker-images-to-quay.io.md) |
-| 13  | [Strimzi canary](./013-kafka-canary.md) |
-| 12  | [Create an Admin Server](./012-admin-server.md) |
-| 11  | [Strimzi UI](./011-strimzi-ui.md) |
-| 10  | [Auth additions for UI and Admin Server](./010-ui-and-admin-server-security.md)  |
-|  9  | [Roadmap to using the CRD v1 API](./009-crd-v1-roadmap.md)  |
-|  8  | [TLS encrypting the Kafka Connect REST API](./008-tls-encrypt-the-kafka-connect-rest-api.md)  |
-|  7  | [Restarting Kafka Connect connectors and tasks](./007-restarting-kafka-connect-connectors-and-tasks.md) |
-|  6  | [Design Documentation](./006-design-docs.md) |
-|  5  | [Improving configurability of Kafka listeners](./005-improving-configurability-of-kafka-listeners.md) |
-|  4  | [GitHub repository restructuring](./004-github-repository-restructuring.md) |
-|  3  | [Remove deprecated Topic Operator deployment in Kafka CRD](./003-remove-deprecated-topic-operator-from-kafka-crd.md) |
-|  2  | [Documentation improvements](./002-documentation-improvements.md) |
-|  1  | [Move Strimzi Kafka operators to Java 11](./001-move-strimzi-kafka-operators-to-java-11.md) |
+| #  | Title                                                                 |
```

Comment:
is the change to the whole file necessary? I see you removed a space from the proposal number column, I would leave it. We are prepared to reach 100 proposals soon ;-)

Reply:
Oops, it was not intentional. Fixed it now.